### ISSUE TITLE
[Feature Request]: Security Improvements for Remotely Save

### ISSUE THREAD

**Issue by sboesen** [remotely-save/remotely-save#396]
### What feature are you suggesting?

Hello, my fork of Remotely Save has some [security improvements](https://github.com/sboesen/remotely-secure#security-updates-from-remotely-save). Please let me know if you're interested in any of these changes (not backwards compatible with existing encrypted vaults) and I'd be happy to open a PR to merge these upstream.

### Ensure no sensitive information

- [X] I ensure that no sensitive information is submitted in the issue.

**Comment by fyears** [comment]
Hi sboesen!

I would like to merge some improvements if applicable.

And also here are some thoughts: https://github.com/sboesen/remotely-sync/discussions/76

**Comment by sboesen** [comment]
Thanks! I responded to that, really appreciate the feedback/details. I think for encryption if you want backwards compatibility I can point to some changes like the salt and possibly the algorithm but I think you have a better grasp of re-encrypting existing files for Remotely Save - Happy to discuss.

Some of the other features (like sync status by @Drumber, modification time sync, etc) needs some further bug tweaks after integrating (could be something I did) but happy to open those separately once ready.
